techmatters/flex-plugins
# CHI1180 Edit contact modal

Primary reviewer: @GPaoloni 

## Description
- Changed the Edit Contact Sections to imitate a modal-like container. With a editingContact state, components are hidden based on when user edits a contact. 
- Added Edit <Section Name> and close button. As such, added these localized strings -  "Contact-EditCaller",
  "Contact-EditChild", "Contact-EditCategories",  "Contact-EditSummary"
- Removed duplicate headers in Contact view with in Case List page

### Checklist
- [n/a] New tests added
- [n/a] Tested for chat contacts
- [n/a] Tested for call contacts

### Related Issues
Fixes # [1180](https://bugs.benetech.org/browse/CHI-1180)

### Verification steps
- Check these four ways to get to a Edit Contact Section
1. From Contacts Queue view, search for a contact and attempt to edit a contact section
2. From Contacts Queue view, search for a case and attempt to edit a contact section
3. From Case List view, open a case, and the contact, and attempt to edit a contact section
4. From Search Contacts/Case, attempt to edit a contact section
![](https://user-images.githubusercontent.com/102122005/176332911-30222c8e-1706-490d-a1ba-ace54589c978.png)
![](https://user-images.githubusercontent.com/102122005/176332917-71264668-b56e-4f43-88d3-7d9b0295d393.png)
